data/state/peerAccount_test.go

```diff
@@ -563,3 +563,93 @@ func TestPeerAccount_DecreaseValidatorSuccessRateWithJournal(t *testing.T) {
 	assert.Equal(t, 1, journalizeCalled)
 	assert.Equal(t, 1, saveAccountCalled)
 }
+
+func TestPeerAccount_IncreaseNumSelectedInSuccessBlocks(t *testing.T) {
+	t.Parallel()
+
+	journalizeCalled := false
+	saveAccountCalled := false
+
+	tracker := &mock.AccountTrackerStub{
+		JournalizeCalled: func(entry state.JournalEntry) {
+			journalizeCalled = true
+		},
+		SaveAccountCalled: func(accountHandler state.AccountHandler) error {
+			saveAccountCalled = true
+			return nil
+		},
+	}
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{}, tracker)
+	err := acc.IncreaseNumSelectedInSuccessBlocks()
+
+	assert.Nil(t, err)
+	assert.True(t, journalizeCalled)
```

Comment:
add check whether the value is the one we want

Reply:
done